src/tools/pub/pubspec.md

```diff
@@ -410,6 +414,37 @@ Each download of the package includes all screenshot files.
 Pub.dev generates the package's thumbnail image from the first screenshot. If 
 this screenshot uses animation, pub.dev uses its first frame.
  
+### Topics
+
+Package authors can use the `topics` field to categorize their package. Topics
+can be used to assist discoverability during search with filters on pub.dev.
+Pub.dev displays the topics on the package page as well as in the search
+results.
+
+The field consists of a list of names. For example:
+
+```yaml
+topics:
+  - network
+  - http
+```
+
+Pub.dev limits topics to the following specifications:
+
+- At most 5 topics per package.
+- The length of the topic name should be 2-32 characters.
+- The topic name should consist of lowercase alphanumeric characters or dash
+  (but no double dash), starting with a-z and ending with a-z or 0-9.
+
+
+When choosing topics it is good practice to look at
+[existing topics]({{site.pub}}/topics) to see if there are established topics
+that match your package.
```

Comment:
**issue:** Write in active voice per [GSG](https://developers.google.com/style/voice).

```suggestion
When choosing topics, first look at
[existing topics]({{site.pub}}/topics) to check if established topics
exist that match your package.
```

Reply:
done

Reply:
We should perhaps consider being less specific here.

Using a word like "may" is or say "may wish to", is perhaps a good idea.

There may be established topics that match your package, but which you don't want to use.
They may be too generic (then they still technically match your package), but it may be that you want to establish a new more specific topic -- that's fine, apply human judgement.